crates/ruff_workspace/src/options.rs

```diff
@@ -1431,6 +1435,39 @@ impl Flake8CopyrightOptions {
     }
 }
 
+/// Options for the `flake8-django` plugin.
+#[derive(
+    Clone, Debug, PartialEq, Eq, Default, Serialize, Deserialize, OptionsMetadata, CombineOptions,
+)]
+#[serde(deny_unknown_fields, rename_all = "kebab-case")]
+#[cfg_attr(feature = "schemars", derive(schemars::JsonSchema))]
+pub struct Flake8DjangoOptions {
+    /// Additional qualified paths to Django URL path functions beyond
+    /// the default `django.urls.path`. This allows the rule to check
+    /// URL patterns defined using custom path functions or re-exported
+    /// path functions from other modules.
+    ///
+    /// For example, if you have a custom module `mytools` that re-exports
+    /// Django's path function, you can add `"mytools.path"` to this list.
+    #[option(
+        default = "[]",
+        value_type = "list[str]",
+        example = r#"
+            # Allow checking URL patterns from custom path functions
+            additional-path-functions = ["mytools.path", "myapp.urls.custom_path"]
+        "#
+    )]
+    pub additional_path_functions: Option<Vec<String>>,
```

Comment:
```suggestion
    pub extend_path_functions: Option<Vec<String>>,
```

I think `extend` is the more typical prefix for these settings. [extend-immutable-calls](https://docs.astral.sh/ruff/settings/#lint_flake8-bugbear_extend-immutable-calls) and [extend-allowed-calls](https://docs.astral.sh/ruff/settings/#lint_flake8-boolean-trap_extend-allowed-calls) are a couple of examples I looked at.